src/jvm/test/org/jetbrains/kotlinx/lincheck_test/trace_debugger/AbstractDeterministicTest.kt

```diff
@@ -45,7 +45,7 @@ abstract class AbstractDeterministicTest {
             ModelCheckingOptions()
                 .actorsBefore(0)
                 .actorsAfter(0)
-                .iterations(30)
+                .iterations(1)
```

Comment:
Some bugs are only reproducible since approximately this number of iterations, so I would just override the value for exact tests using `customize`. And I would recommend setting it to at least `2`.

Reply:
`iterations` parameter and data structures API shouldn't be used in these tests in the first place.